jenkins/deploy-clio-server.sh

```diff
@@ -1,210 +1,256 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Is there a case where `bash` isn't at `/bin/bash`, or we want to execute a different `bash` than the default one?

Reply:
the location of bash will probably not change, but `/usr/bin/env bash` is the accepted standard for how to denote a bash script.